integration_tests/e2e/goal/createGoals.cy.ts

```diff
@@ -237,14 +237,16 @@ context('Create goals', () => {
         .withRequestBody(
           matchingJsonPath(
             '$[?(@.goals.size() == 2 && ' +
-              "@.goals[0].prisonNumber == 'G6115VJ' && " +
+              "@.goals[0].prisonId == 'MDI' && " +
```

Comment:
`prisonNumber` is no longer in the request body object (and should never have been)
`prisonId` and `targetCompletionDate` were always in the request body but were never tested here.

Reply:
I find prisonNumber to be a bad name anyway. You have prisonNumber and prisonId. One refers to a person, one refers to a place 🙃

Reply:
Interesting you should say that; we've never liked the name `prisonNumber` either! 
But AFAIK it's the "official" / standard HMPPS terminology for this data item (I got this from Mike Halma when I worked with him here on a previous project)
From the outset we wanted to use the "correct" terminology for things like this, rather than using "un-official" variants such as prisonerId, prisonerNumber, offenderId, pnc etc etc etc
🤷‍♂️